README.md

```diff
@@ -90,7 +90,7 @@ you prefer to only receive updates for tagged releases, use
 
 ## Contributing
 
-See the [CONTRIBUTING.md][] document.
```

Comment:
whoops, works in some flavors.

I submitted 1ca9af77af79dfcee3811e6ffba60e53859113c2 to clean things up.